app/graphql/types/query_type.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Types
+  class QueryType < Types::BaseObject
+    field :node, Types::NodeType, null: true, description: "Fetches an object given its ID." do
+      argument :id, ID, required: true, description: "ID of the object."
+    end
+
+    def node(id:)
+      context.schema.object_from_id(id, context)
+    end
+
+    field :nodes, [Types::NodeType, { null: true }], null: true, description: "Fetches a list of objects given a list of IDs." do
+      argument :ids, [ID], required: true, description: "IDs of the objects."
+    end
+
+    def nodes(ids:)
+      ids.map { |id| context.schema.object_from_id(id, context) }
+    end
+
+    # Add root-level fields here.
+    # They will be entry points for queries on your schema.
```

Comment:
This is deleted in the next PR but we could get rid of it here instead

Reply:
I agree that removing it here seems cleaner and will make the git history easier to follow.